src/components/Projects/MyProjects.tsx

```diff
@@ -0,0 +1,68 @@
+import { t, Trans } from '@lingui/macro'
+import Grid from 'components/shared/Grid'
+import Loading from 'components/shared/Loading'
+import ProjectCard from 'components/shared/ProjectCard'
+import { NetworkContext } from 'contexts/networkContext'
+import { ThemeContext } from 'contexts/themeContext'
+import { useMyProjectsQuery } from 'hooks/v1/Projects'
+import React, { useContext } from 'react'
+import { InfoCircleOutlined } from '@ant-design/icons'
+
+export default function MyProjects() {
+  const { userAddress } = useContext(NetworkContext)
+
+  const { data: projects, isLoading } = useMyProjectsQuery(userAddress)
+
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  return (
```

Comment:
**suggestion** could we add a state here that prompts the user to connect their wallet if `userAddress` is falsey (haven't connected a wallet)? WDYT?

Reply:
Yeah I like it, and we could do the same for the holdings tab.